cf-usb/config-sle-production-master.yaml

```diff
@@ -0,0 +1,31 @@
+src-repo: https://github.com/SUSE/cf-usb-sidecar.git
+src-branch: master
+
+ci-repo: https://github.com/SUSE/cf-ci.git
+ci-branch: master
+
+docker-tag: latest
+docker-server: *docker-internal-registry
+docker-username: *docker-internal-username
+docker-password: *docker-internal-password
+
+docker-cert-domain: *docker-public-registry
+docker-cert-ca: *ca-cert-suse
+
+buildbase-base-image:              *docker-image-sles
+buildbase-repo-cloud-tools:        *zypper-repo-sle-server
+buildbase-repo-devel-languages-go: *zypper-repo-sle-sdk
+buildbase-repo-extra:              *zypper-repo-ibs-ga
+
+charts-release-tool-url: *cap-release-tool-url
```

Comment:
I don't see these anchors defined anywhere.

Reply:
concourse secrets ?

Reply:
The value is defined, but there's no anchor. I'm seeing:

`cap-release-tool-url: "http://example.com"` when I'm expecting to see

`cap-release-tool-url: &cap-release-tool-url "http://example.com"`

Reply:
Fixed in SUSE/cloudfoundry@6c68fcc249bd55cee7ff30c8cd3ffbccfcec01cf